reference.md

```diff
@@ -858,56 +858,70 @@ The following fields are all attributes within the main "data" object for this f
 ```
 
 ### realtime_booking
+This dynamic query provides time/cost estimates and realtime_booking information for specific location. 
```

Comment:
Just added some extra description to make it clear why `realtime_booking` and `wait_time` both exist.

```suggestion
This dynamic query provides time/cost estimates and realtime_booking information for specific location. It can be used in situations where a provider expects to have more fine-grained or accurate booking information for a given trip, compared to what is available via other endpoints.

For example, a provider may be able to give more accurate pricing information for a specific trip, compared to the broader rules-based pricing information available from `fares.json`.
```